### PR TITLE
Refs #28068 - Include dynflow-sidekiq.rb

### PR DESCRIPTION
**Pull request by ekohl**
This file is needed to start dynflow-sidekiq@.service since it's mentioned as a --require path.

### CODE REVIEW

File: debian/bionic/foreman/foreman-dynflow-sidekiq.install [hunk 1]
```diff
@@ -1,3 +1,4 @@
+extras/dynflow-sidekiq.rb /usr/share/foreman/extras/dynflow-sidekiq.rb
```
Comment (evgeni) [inline debian/bionic/foreman/foreman-dynflow-sidekiq.install]:
```suggestion
extras/dynflow-sidekiq.rb /usr/share/foreman/extras
```

dest is a folder. not the path. same in the other file

**Comment by ekohl** [comment]
So like this?

**Comment by evgeni** [comment]
Exactly

**Comment by ekohl** [comment]
It's there now:

```
# dpkg -L foreman-dynflow-sidekiq
/.
/etc
/etc/foreman
/etc/foreman/dynflow
/etc/foreman/dynflow/orchestrator.yml
/etc/foreman/dynflow/worker.yml
/lib
/lib/systemd
/lib/systemd/system
/lib/systemd/system/dynflow-sidekiq@.service
/usr
/usr/share
/usr/share/doc
/usr/share/doc/foreman-dynflow-sidekiq
/usr/share/doc/foreman-dynflow-sidekiq/changelog.Debian.gz
/usr/share/doc/foreman-dynflow-sidekiq/changelog.gz
/usr/share/doc/foreman-dynflow-sidekiq/copyright
/usr/share/foreman
/usr/share/foreman/bundler.d
/usr/share/foreman/bundler.d/dynflow_sidekiq.rb
/usr/share/foreman/extras
/usr/share/foreman/extras/dynflow-sidekiq.rb
```

**Pull request merged**